src/state/queries/video/video-upload.web.ts

```diff
@@ -22,53 +22,56 @@ export const useUploadVideoMutation = ({
 
   return useMutation({
     mutationFn: async (video: CompressedVideo) => {
-      const uri = createVideoEndpointUrl('/upload', {
+      const uri = createVideoEndpointUrl('/xrpc/app.bsky.video.uploadVideo', {
         did: currentAccount!.did,
-        name: `${nanoid(12)}.mp4`, // @TODO what are we limiting this to?
+        name: `${nanoid(12)}.mp4`, // @TODO: make sure it's always mp4'
```

Comment:
can remove this todo if you want

Reply:
I'll do it in another PR because I'm not actually preventing MOVs yet

Reply:
oh! got it